entwatch/ze_magmadrake_b5.cfg

```diff
@@ -0,0 +1,104 @@
+"entities"
+{
+	"0"
+	{
+		"name"            "human_Piggyback"
+		"shortname"       "Piggyback" 
+		"color"           "{default}" 
+		"buttonclass"     "func_button"
+		"allowtransfer"   "true"
+		"forcedrop"       "false"
+		"hud"             "true"
+		"mode"            "0"
+		"cooldown"        "0" 
+		"maxuses"         "0"
+		"hammerid"        "11111018"
+		"buttonid"        "11111118"
+	}
+	"1"
+	{
+		"name"            "human_HealOrb"
+		"shortname"       "Heal Orb" 
+		"color"           "{default}" 
+		"buttonclass"     "func_button"
+		"allowtransfer"   "true"
+		"forcedrop"       "true"
+		"hud"             "true"
+		"mode"            "3"
+		"maxuses"         "1" 
+		"hammerid"        "1111103"
+		"buttonid"        "1111113"
+	}
+	"2"
+	{
+		"name"				"human_Magmapush"
+		"shortname"			"Magma Push"
+		"color"				"{default}"
+		"buttonclass"		"func_button"
+		"allowtransfer"		"true"
+		"forcedrop"			"true"
+		"hud"				"true"
+		"mode"				"2"
+		"cooldown"			"50"
+		"hammerid"			"136492"
+		"buttonid"			"136494"
+	}
+	"3"
+	{
+		"name"				"human_DragonSlayer"
+		"shortname"			"DragonSlayer"
+		"color"				"{default}"
+		"buttonclass"		"func_button"
+		"allowtransfer"		"true"
+		"forcedrop"			"true"
+		"hud"				"true"
+		"mode"				"0"
+		"hammerid"			"11111030"
+		"buttonid"			"11111130"
+		
+	}
+	"4"
+	{
+
+		"name"				"zm_Flame"
+		"shortname"			"Flame"
+		"color"				"{default}"
+		"buttonclass"		"func_button"
+		"allowtransfer"		"true"
+		"forcedrop"			"true"
+		"hud"				"true"
+		"mode"				"2·"

```

Comment:
The period in here is not valid

Reply:
I did it at night. I got a little dizzy @_@
I thought I could omit some fields because I thought they would have a default value. It worked fine when I tested it